git_machete/github.py

```diff
@@ -24,9 +25,35 @@ def __repr__(self) -> str:
 GITHUB_TOKEN_ENV_VAR = 'GITHUB_TOKEN'
 
 
-def github_token() -> Optional[str]:
+def _token_from_gh() -> Optional[str]:
+    proc = subprocess.run(
+        ["gh", "auth", "status", "--show-token"],
+        stdout=subprocess.PIPE, stderr=subprocess.PIPE
+    )
+
```

Comment:
Btw I'm getting:

```
$ gh auth status --show-token
unknown command "auth" for "gh"
```

but I've probably got some ancient version of `gh`... lemme see...

Reply:
`A new release of gh is available: 0.10.0 → v1.11.0` 😅 

Anyway, you can mention somewhere that at least (some version) of `gh` is required... I suppose `1.0`?